### PR TITLE
Adds third batch of 4-10 RNs

### PR DESCRIPTION
**Pull request by stevsmit**
Bug doc text batch 3

Preview: https://deploy-preview-42151--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes

No QE needed.

Can be merged upon approval.

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -703,6 +703,21 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
...
+
+* Previously, an old version of the Kubernetes client library was present in the `baremetal-runtimecfg` project. When a Virtual IP (VIP) failed, the client connections might not be closed in a timely fashion. This could result in long delays for monitor containers that rely on talking to the API. This fix updates the client library. Now, connections are closed as expected on VIP failovers and the monitor containers do not hang for an excessive period of time.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1995021[*BZ#1995021*])
+
+* Previously, in clusters that use Stateless Address AutoConfiguration (SLAAC), the Ironic `addr-gen-mode` parameter was not being persisted to the OVNKubernetes bridge. As a result, the IPv6 addresses could change when the bridge was created. This broke the cluster because node IP changes are unsupported. This fix persists the `addr-gen-mode` parameter when creating the bridge. The IP address is now consistent throughout the deployment process.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1990625[*BZ#1990625*])"
```
Comment (ahardin-rh) [inline release_notes/ocp-4-10-release-notes.adoc:715]: possible unnecessary line break before the bug link

**Comment by ahardin-rh** [comment]
Just a few minor comments. Thanks!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -703,6 +703,21 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
...
+[id="ocp-4-10-machine-config-operator-bug-fixes"]
+==== Machine Config Operator
+
+* Previously, a warning message sent by the `clone_slave_connection` function was incorrectly stored in a `new_uuid` variable, which is intended to store only the connection's UUID. As a result, `nmcli` commands that include the `new_uuid` variable were failing due to the incorrect value being stored in the `new_uuid` variable. With this fix, the `clone_slave_connection` function warning message is redirected to `stderr`. Now, `nmcli` commands that reference the `new_uuid` variable do not fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2022646[*BZ#2022646*])
+
+* Previously, an old version of the Kubernetes client library was present in the `baremetal-runtimecfg` project. When a Virtual IP (VIP) failed, the client connections might not be closed in a timely fashion. This could result in long delays for monitor containers that rely on talking to the API. This fix updates the client library. Now, connections are closed as expected on VIP failovers and the monitor containers do not hang for an excessive period of time.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1995021[*BZ#1995021*])
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply release_notes/ocp-4-10-release-notes.adoc:712]: ty

**Pull request merged**